README.md

```diff
@@ -11,6 +11,8 @@ Material Components for the web is the successor to [Material Design Lite](https
 
 MDC-Web strives to seamlessly incorporate into a wider range of usage contexts, from simple static websites to complex, JavaScript-heavy applications to hybrid client/server rendering systems. In short, whether you're already heavily invested in another framework or not, it should be easy to incorporate Material Components into your site in a lightweight, idiomatic fashion.
 
+> **Note:** MDC-Web is intended to be used for other developers to execute Material Design properly. MDC-Web delegates higher level capabilities to other JS frameworks and libraries. An example is that MDC-Web neither use web components nor renders the DOM of its components. By doing low-level operations, MDC-Web its able to accomplish its goals.
```

Comment:
More feedback:
- I don't think this has to be quoted, nor does it need `**Note:**`. It can just be a normal paragraph
- "higher-level capabilities" is a bit ambiguous; how about instead we say something like "MDC-Web does not handle things like DOM rendering, and automatic component instantiation". I don't want to use the word "delegate" because I don't want it to sound like people who want to use the library are expected to do a whole bunch of work just to get it working properly. 
- We can remove the last sentence. We don't do "low-level" operations, we just avoid doing certain things that we feel would step on the toes of non-UI-library code.